src/clj_foundation/errors.clj

```diff
@@ -109,3 +106,65 @@ attempts, the last caught exception is re-thrown."
   The remaining parameters (if any) are that function's arguments."
   [tries pause-millis]
   (fn [f & args] (apply retry tries pause-millis f args)))
+
+
+(s/defn replace-if :- s/Any
```

Comment:
Where would you suggest (and they've been in errors.clj for a year now)?

Rationale: Most are idioms for dealing with boundary condition checking.